src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -68,6 +68,9 @@
     </Page>
     <Page Include="UI\ServerSelection\ServerSelectionWindow.xaml" />
     <Page Include="UI\ManageConnections\ManageConnectionsWindow.xaml" />
+    <Page Include="UI\ConnectionInfo.xaml">
+      <Generator>MSBuild:Compile</Generator>
```

Comment:
why did it only add that for this xaml? 🤔

Reply:
Because VS works in mysterious ways. I can manually remove it to make ti consistent. In the end, I don't think it makes any real difference.

Reply:
Don't need to remove it. I was just curious